pkgs/os-specific/darwin/apple-source-releases/bootstrap_cmds/default.nix

```diff
@@ -3,27 +3,18 @@
 appleDerivation {
   nativeBuildInputs = [ yacc flex ];
 
-  buildPhase = ''
+  patchPhase = ''
     cd migcom.tproj
+
+    # redundant file, don't know why apple not removing it.
+    rm handler.c
```

Comment:
Do we have to remove this?

Reply:
Yes, we use `cc *.c` below, this file can't even be compiled, we skipped it previous to this PR too.